core/include/kernel/tee_time.h

```diff
@@ -38,5 +38,7 @@ TEE_Result tee_time_get_ta_time(const TEE_UUID *uuid, TEE_Time *time);
 TEE_Result tee_time_get_ree_time(TEE_Time *time);
 TEE_Result tee_time_set_ta_time(const TEE_UUID *uuid, const TEE_Time *time);
 void tee_time_wait(uint32_t milliseconds_delay);
```

Comment:
Let's cut the pear in two (french expression :) : what about `tee_busy_mdelay()` ?